### PR TITLE
core/anal: complete the list of analhint commands to print

### PR DESCRIPTION
**Pull request by ret2libc**

### CODE REVIEW

File: libr/core/anal.c [hunk 1]
```diff
@@ -541,29 +541,34 @@ static int cb(void *p, const char *k, const char *v) {
 	case '*':
 		HINTCMD_ADDR (hint, arch, "aha %s");
 		HINTCMD_ADDR (hint, bits, "ahb %d");
-		HINTCMD_ADDR (hint, size, "ahl %d");
+		HINTCMD_ADDR (hint, size, "ahs %d");
 		HINTCMD_ADDR (hint, opcode, "aho %s");
-		HINTCMD_ADDR (hint, opcode, "ahs %s");
```
Comment (ret2libc) [inline libr/core/anal.c]: Was there any reason to have "opcode" duplicated in this and in the next line, but with different ```ah``` commands?

**Comment by ret2libc** [comment]
With this, also immbase information should be stored into the project file

**Comment by Maijin** [comment]
May close #3767

**Comment by radare** [comment]
Merged!

**Pull request closed without merging**